<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetingWeather</class>
 <widget class="QWidget" name="SetingWeather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>set weather</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/xml/scrpis/Humidity.png</normaloff>:/xml/scrpis/Humidity.png</iconset>
  </property>
  <widget class="QGroupBox" name="AreaSelectGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>356</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string>Zone and Area</string>
   </property>
   <widget class="QComboBox" name="ZoneBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="AreaBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ChooseCity</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="AreaSelectedCityLabel">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>34</y>
      <width>129</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Zonelabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>44</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>74</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ViewSelectGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>356</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string>View</string>
   </property>
   <widget class="QSlider" name="WIndowOpacitySlider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>192</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="WindowOpacityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Window Opacity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WindowColorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Window Color</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="NowColorLabel">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>85</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewChoseColorButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Chose</string>
    </property>
   </widget>
   <widget class="QLabel" name="SelectedColorLabel">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>85</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="WIndowSliderValueLabel">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="transparentBox">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>86</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>transparent</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SetingWeather</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AreaBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>AreaSelectedCityLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WIndowOpacitySlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>WIndowSliderValueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparentBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ViewChoseColorButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
